.circleci/config.yml

```diff
@@ -14,6 +14,9 @@ jobs:
     steps:
       - checkout
 
+      # Make sure we are on the latest npm version
+      - run: npm i -g npm
```

Comment:
Should we not just make sure we're on a specific version of npm rather than the generic latest?

Reply:
@treygriffith ill stick it to a major